app/javascript/src/controller_pages.js

```diff
@@ -32,7 +32,8 @@ const TextQuestion = require('./question_text');
 const TextareaQuestion = require('./question_textarea');
 const AutocompleteQuestion = require('./question_autocomplete');
 
-const DialogConfiguration = require('./component_dialog_configuration');
+// const DialogConfiguration = require('./component_dialog_configuration');
+const Dialog = require('./component_dialog');
 const DialogApiRequest = require('./component_dialog_api_request');
```

Comment:
I was getting the impression DialogForm was replacing the DialogApiRequest components.
Am I wrong or should this line (and presumably related file) be removed?

Reply:
No.  `DialogForm` came first.  In doing this work, I realised some of the `DialogApiRequest` calls we were making actually had forms in the returned templates so it made more sense, and gave more consistency for them to become `DialogForm` calls 

Especially as that meant the DialogForm class handles the submitting whereas with the `DialogApiRequest` the submitting was having to be handled in the `#build` callback event.